test/name_service/api.clj

```diff
@@ -32,4 +35,72 @@
          "X-XSS-Protection" "1; mode=block"
          "X-Download-Options" "noopen"
          "X-Permitted-Cross-Domain-Policies" "none"
-         "Content-Security-Policy" "object-src 'none'; script-src 'unsafe-inline' 'unsafe-eval' 'strict-dynamic' https: http:;"})))
\ No newline at end of file
+         "Content-Security-Policy" "object-src 'none'; script-src 'unsafe-inline' 'unsafe-eval' 'strict-dynamic' https: http:;"})))
+
+(deftest test-delete-keymap-wrong-params
```

Comment:
If you add route name you can use it instead of path in your tests like this:
```clj
[io.pedestal.http.route :as route]

...

(def url-for
  (route/url-for-routes service/routes))
...
(response-for service
              :delete (url-for <some-route-name>)
              ...
```